payment/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from payment.models import Payment
+
+
+class PaymentSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Payment
+        fields = (
+            "id",
+            "status_payment",
+            "type_payment",
+            "borrowing",
+            "session_url",
+            "session_id",
+            "money",
+        )
+        read_only_fields = ("id", "money",)
```

Comment:
But the `money` field should also be calculated somewhere. Or are you going to implement this in some of the following tasks?

Reply:
in next task)
I understand that in this task i need implement only model/serializer/views